### PR TITLE
[ADD] Towel wrap

### PR DESCRIPTION
**Pull request by SandrinePDR**
## References

_None_

## About The Pull Request

Added a towel that can be wrapped around the body or a character's hair

## Changelog

Authored by: Taja & Sandrine

```md
Assets:
- [Towel Wrap] New asset either for the body (worn as a dress) or the head (worn as a hat)
```

## Checklist

- [X] The change has been tested locally
- [X] Licensing information and credits were filled in/modified for added/modified assets
- [X] I understand this change is licensed based on [Pandora's Asset Licensing](https://github.com/Project-Pandora-Game/pandora-assets/blob/master/ASSET_LICENSING.md) information

### CODE REVIEW

**Review by ClaudiaMia — COMMENTED** [review]
Amazing new asset! We definitely had a strong need for that! 

I feel it works well enough with the two size variants, but I do see some slight glitching around the arm pits in some poses back/front view due to the different sizes of the front and back images and how they overlap.

Would it be okay for you and taja if I had a look tomorrow and tried to separate a breast layer from the images to try and do the breast scaling more form fitting?

**Comment by SandrinePDR** [comment]
Absolutely

File: src/assets/dresses/towel_wrap/towel_wrap.asset.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	ownership: {
+		responsibleContributor: 'Sandrine <118102950+SandrinePDR@users.noreply.github.com>',
+		credits: ['Taja'],
+		modificationPolicy: `Fixes and New uses, otherwise ask`,
+		reusePolicy: 'Ask first',
+		licensing: [
+			{
+				part: 'towel',
+				source: 'Self-Made',
+				copyrightHolder: 'Tajo',
```
Comment (Jomshir98) [inline src/assets/dresses/towel_wrap/towel_wrap.asset.ts:63]: Is this intentional or should it also be `Taja` as in credits?

File: src/assets/headwear/towel_wrap/towel_wrap.asset.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	ownership: {
+		responsibleContributor: 'Sandrine <118102950+SandrinePDR@users.noreply.github.com>',
+		credits: ['Taja'],
+		modificationPolicy: `Fixes and New uses, otherwise ask`,
+		reusePolicy: 'Ask first',
+		licensing: [
+			{
+				part: 'towel',
+				source: 'Self-Made',
+				copyrightHolder: 'Tajo',
```
Comment (Jomshir98) [inline src/assets/headwear/towel_wrap/towel_wrap.asset.ts:33]: Is this intentional or should it also be `Taja` as in credits?

File: src/assets/dresses/towel_wrap/towel_wrap.asset.ts (hunk 1 above)
Comment (ClaudiaMia) [reply src/assets/dresses/towel_wrap/towel_wrap.asset.ts]: Clarified with Sandrine that it is a typo and I fixed it.

File: src/assets/headwear/towel_wrap/towel_wrap.asset.ts (hunk 2 above)
Comment (ClaudiaMia) [reply src/assets/headwear/towel_wrap/towel_wrap.asset.ts:33]: Clarified with Sandrine that it is a typo and I fixed it.

Comment (SandrinePDR) [reply src/assets/headwear/towel_wrap/towel_wrap.asset.ts]: Taja and I are fine with your fixes @ClaudiaMia

**Review by ClaudiaMia — APPROVED** [review]

**Pull request merged**